docs/release-notes/release-notes-0.19.0.md

```diff
@@ -224,6 +224,24 @@ The underlying functionality between those two options remain the same.
 
 
 ## Breaking Changes
```

Comment:
Instead of putting these changes under the `Breaking Changes` heading, can we add a new heading `Deprecations` and put it under that?

The change would be breaking when we finally remove these rpcs in 0.21